build/python/build/libs.py

```diff
@@ -101,9 +101,9 @@
 )
 
 wolfssl = AutotoolsProject(
+    'http://www.caztech.com/glider/xcsoar/skysight/wolfssl-4.0.0.tar.gz',
     'https://fossies.org/linux/misc/wolfssl-4.0.0.tar.gz',
-    'https://github.com/wolfSSL/wolfssl/archive/v4.0.0-stable.tar.gz',
-    '372bfe2a6ddeb2f42f1256ee084bb8c0575dd7323db3990cb2658e9924dc58be',
+    '0da9104f9c442d8a4ec1fbbba9db73bb',
```

Comment:
Why, and why a new hash?

Reply:
The reason why I separate is to clearly indicate the previous commit is written by jfwells and I update its problem. I'll integrate this patch into previous one.

Reply:
But why use your server to download it?
And why is the hash changed? Are you secretly injecting a modified version of the original tarball?

Reply:
Woflssl is available from github. I would tag to the stable branch for 4.0.0:
https://github.com/wolfSSL/wolfssl/archive/v4.0.0-stable.zip

Reply:
The file name of github is v4.0.0-stable.tar.gz which does not comply with file name convention in project.py.
fossies.org has wolfssl-4.2.0.zip while it no longer keeps wolfssl-4.0.0.tar.gz. project.py does not handle zip file while suppose to handle.
m = re.match(r'^(.+)\.(tar(\.(gz|bz2|xz|lzma))?|zip)$', basebname)
I am blocked.
BTW, I did not modify anything. I just copy v4.0.0-stable.tar.gz to my website with changing its name.